UnityProject/Assets/Standard Assets/Logger/ThreadLogger.cs

```diff
@@ -0,0 +1,30 @@
+using System.Collections.Generic;
+using UnityEngine;
+
+
+public class ThreadLogger : MonoBehaviour
+{
+	public static List<string> otherThreadLogs = new List<string>();
+
+	// Update is called once per frame
+	void Update()
+	{
+		lock (otherThreadLogs)
+		{
+			foreach (var log in otherThreadLogs)
+			{
+				Logger.LogError(log);
+			}
+
+			otherThreadLogs.Clear();
```

Comment:
does this have any impact on performance? Clearing a list every frame

Reply:
this was already in the project just moved

Reply:
aight